package.json

```diff
@@ -54,7 +54,8 @@
     "@angular/platform-browser": ">=6.x <=8.x"
   },
   "dependencies": {
-    "ng-mocks": "^7.8.0"
+    "ng-mocks": "^7.8.0",
+    "rxjs": "^6.3.3"
```

Comment:
rxjs should be part of devDependencies, not main dependencies.

Reply:
> rxjs should be part of devDependencies, not main dependencies.

right, that wasn't too smart. I have fixed it by disabling tslint in rxjs import inside one of the test.